FactoryWeather/App/VIP Home/HomeViewController.swift

```diff
@@ -0,0 +1,103 @@
+//
+//  HomeViewController.swift
+//  FactoryWeather
+//
+//  Created by Matej Korman on 08/01/2019.
+//  Copyright © 2019 Matej Korman. All rights reserved.
+//
+
+import SnapKit
+
+protocol HomeDisplayLogic: class {
+    func displayWeather(_ weather: HomeDataSource, forLocation location: Location)
+    func searchTextFieldIsHidden(_ isHidden: Bool)
```

Comment:
We could improve the naming on this one a bit, so we're more in line with the `displayX` naming for the `DisplayLogic`. Of course, we should not force the `displayX` naming for situations where it's not really appropriate.

```suggestion
    func displaySearchTextField(hidden: Bool)
```

Reply:
Now since we concluded that this method is used only within `HomeSceenLogic` does this new naming make sense or it should remain the same.

Reply:
I would rename to something more action-like - `setSearchTextFieldHidden(_ hidden: Bool)`. Usually, we use the `isX` pattern for properties, since the semantics actually describe a state, and we use `setX` for methods.

Reply:
Ok, I agree. The naming seamed a bit strange to me too, but I wasn't able to find a better one.

Reply:
Not a problem. 

When thinking about naming, you can keep in mind the following:
> Usually, we use the isX pattern for properties, since the semantics actually describe a state, and we use setX for methods.